### ISSUE TITLE
Calendar needs two clicks for first selection on touch devices

### ISSUE THREAD

**Issue by tpinne** [grommet/grommet#4150]
### Expected Behavior

Selecting the first date on touch devices should focus the calendar and select the clicked date in one step, as it does on non touch devices.

### Actual Behavior

First touch (click) focuses the component. Second touch (click) selects the date.

### URL, screen shot, or Codepen exhibiting the issue

![Bildschirmvideo aufnehmen 2020-06-05 um 13 16 58](https://user-images.githubusercontent.com/1384345/83871072-aca4bb80-a72f-11ea-99d8-25bc965d27e0.gif)

### Steps to Reproduce

Use a calendar on desktop browser. Then use developer tools to simulate a touch device or use real one, reload page and repeat selecting a date.

**Comment by halocline** [comment]
Hi. Using this [CodeSandbox](https://codesandbox.io/s/relaxed-dijkstra-3i980?file=/src/App.js) running Grommet 2.15.0, I have been unable to reproduce this issue.

I have tested manually on iPhone Safari, Chrome, and Firefox. All selected the date on first touch.

I also tested using mobile simulations with DevTools on Desktop. All selected the date on first touch.

Here are a couple screen recordings:

![issue-4150-mobile-chrome](https://user-images.githubusercontent.com/1756948/95265048-a6157600-07ed-11eb-910e-7a8327a1a9d1.gif)
![issue-4150-mobile-ff](https://user-images.githubusercontent.com/1756948/95265054-a746a300-07ed-11eb-8d22-b6956cd4cb5c.gif)

@tpinne Are you still experiencing this issue using Grommet 2.15.0?
